corehq/messaging/smsbackends/airtel_tcl/tests/test_airtel_tcl.py

```diff
@@ -110,14 +110,18 @@ def test_get_url(self):
         )
 
     def test_get_phone_number(self):
-        self.assertEqual(AirtelTCLBackend.get_phone_number('+910123456789'), '0123456789')
-        self.assertEqual(AirtelTCLBackend.get_phone_number('910123456789'), '0123456789')
+        self.assertEqual(AirtelTCLBackend.get_phone_number('+918123456789'), '8123456789')
+        self.assertEqual(AirtelTCLBackend.get_phone_number('916123456789'), '6123456789')
 
         with self.assertRaises(InvalidDestinationNumber):
-            AirtelTCLBackend.get_phone_number('+999123456')
+            AirtelTCLBackend.get_phone_number('+9959123456')
```

Comment:
why did we need to change this? it was a test for a number with incorrect length and not starting with 91.